### PR TITLE
Reference popout by name persona dialog

### PR DESCRIPTION
**Pull request by pymagda**
Issue #21

### CODE REVIEW

File: browserid/pages/sign_in.py [hunk 1]
```diff
@@ -42,15 +42,11 @@ class SignIn(Base):
...
-            for handle in self.selenium.window_handles:
-                self.selenium.switch_to_window(handle)
-                WebDriverWait(self.selenium, self.timeout).until(lambda s: s.title)
-                if self.selenium.title == self._page_title:
-                    self._sign_in_window_handle = handle
-                    break
-            else:
-                raise Exception('Popup has not loaded')
+        self.selenium.switch_to_window('__persona_dialog')
+        if self.selenium.title.startswith('Mozilla Persona'):
```
Comment (davehunt) [inline browserid/pages/sign_in.py]: We should be able to maintain the original condition: `self.selenium.title == self._page_title`

File: browserid/pages/sign_in.py [hunk 2]
```diff
@@ -42,15 +42,11 @@ class SignIn(Base):
...
+            self._sign_in_window_handle = '__persona_dialog'
```
Comment (davehunt) [inline browserid/pages/sign_in.py]: This needs to be a window handle, not a string. You can use: `self._sign_in_window_handle = selenium.current_window_handle`

**Comment by davehunt** [comment]
Thanks for the patch @pymagda! With a couple of fixes this will be ready to merge.

**Comment by rbillings** [comment]
I'm closing the original issue as BIDPOM is going away in November 2016, and this issue has not been updated in a long time.

**Pull request closed without merging**